cxx-qt-gen/src/gen_rs.rs

```diff
@@ -111,25 +114,33 @@ impl RustType for QtTypes {
             Self::QSize => quote! {cxx_qt_lib::QSize},
             Self::QSizeF => quote! {cxx_qt_lib::QSizeF},
             Self::Str => match target_type {
-                TargetType::Cpp => quote! {cxx_qt_lib::QStringCpp},
+                TargetType::Cpp => quote! {cxx_qt_lib::QString},
                 TargetType::Rust => quote! {str},
             },
             Self::String => match target_type {
-                TargetType::Cpp => quote! {cxx_qt_lib::QStringCpp},
+                TargetType::Cpp => quote! {cxx_qt_lib::QString},
                 TargetType::Rust => quote! {String},
             },
+            Self::QString => match target_type {
+                TargetType::Cpp => quote! {cxx_qt_lib::QString},
+                TargetType::Rust => quote! {cxx_qt_lib::QString},
+            },
             Self::QTime => quote! {cxx_qt_lib::QTime},
             Self::QUrl => match target_type {
-                TargetType::Cpp => quote! {cxx_qt_lib::QUrlCpp},
+                TargetType::Cpp => quote! {cxx_qt_lib::QUrl},
                 TargetType::Rust => quote! {cxx_qt_lib::QUrl},
             },
             Self::QVariant => match target_type {
-                TargetType::Cpp => quote! {cxx_qt_lib::QVariantCpp},
+                TargetType::Cpp => quote! {cxx_qt_lib::QVariant},
                 TargetType::Rust => quote! {cxx_qt_lib::QVariant},
             },
             Self::U8 => quote! {u8},
             Self::U16 => quote! {u16},
             Self::U32 => quote! {u32},
+            Self::UniquePtr { inner } => {
+                let inner = inner.cxx_qt_lib_type(target_type);
+                quote! {UniquePtr<#inner> }
```

Comment:
I think the trailing space should actually be removed to match the others :-) ... i think because it's a macro rustfmt doesn't pick it up.

If we put spaces on either side then all of them should have it, but hopefully this code will go away in the future anyway :-)